src/CGB_Registers.md

```diff
@@ -219,13 +219,40 @@ ON/OFF pulses (length 10us ON, 17.5us OFF each) instead of a permanent
 880us LED ON signal. Even though being generally CGB compatible, the GBA
 does not include an infra-red port.
 
+### FF4C — KEY0 (CGB Mode only): CPU mode select 
+
+This GBC-only register (which is not officaly documented) is written only by the CGB boot ROM,
+as it gets locked after the bootrom finish execution (by a write to the [BANK register](<#Monochrome models (DMG0, DMG, MGB)>)).
+
+Once it is locked, the behavior of the system can't be changed without a reset (This behavior can be observed using [this test ROM](https://github.com/alloncm/MagenTests?tab=readme-ov-file#key0-cpu-mode-register-lock-after-boot)).
+
+As a result of the above most of the behavior is not directly testable without hardware manipulation.
+Even though we can't test it's behavior directly we can inspect the disassembly of the CGB bootrom and infer the following: 
+
+{{#bits 8 >
+   "KEY0" 2:"DMG compatibility mode"
+}}
+
+- **DMG compatibility mode**: `0` = Disabled (full CGB mode, for regular CGB cartridges), `1` = Enabled (for DMG only cartridges)
+
+#### Bit 3 and "PGB mode"
+
+It has been speculated based on leaked documents that setting bit 3 is related to a special mode called "PGB" for controlling the LCD externally.
```

Comment:
I would move this inside the "research needed" boxes and avoid mentioning where the speculation comes from. I'd make this generic and link from the Research Needed box an issue here on github where we can discuss / contribute to this specific topic

Reply:
What's the reason to mention bit 3 at all anyway?

Reply:
Yeah, exactly, I don't see a reason for it at the moment. We can have the speculation/discussions in a issue and simply link to it

Reply:
Since other sections (like the OPRI register) mentions PGB mode with the values activating it I thought it will help to avoid confusion regarding those values.

Reply:
Just pushed new changes and opened a new issue addressing most of the comments except for bit 3 so Im not resolving this yet.
Let me know what you think regarding the other sections mentioning PGB mode values.